### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by hoc081098**
#27

### CODE REVIEW

File: example/pubspec.yaml [hunk 1]
```diff
@@ -4,19 +4,19 @@ publish_to: none
...
+  flutter: '>=3.22.2'
 
 dependencies:
   flutter:
     sdk: flutter
   flutter_google_places_hoc081098:
     path: ../
-  google_api_headers: ^2.0.0
-  uuid: ^3.0.7
+  google_api_headers: ^4.1.2
```
Comment (devsideal) [inline example/pubspec.yaml:15]: I dont thing, we need this package in this example.

Comment (hoc081098) [reply example/pubspec.yaml]: @devsideal ,
google_api_headers is also used in the example 🙏 
https://github.com/hoc081098/flutter_google_places_hoc081098/blob/df1d04ade493aa1cbbd0a81aa09353d858498764/example/lib/main.dart#L144

File: example/pubspec.yaml [hunk 2]
```diff
@@ -4,19 +4,19 @@ publish_to: none
 description: A new Flutter project.
 
 environment:
-  sdk: '>=2.14.0 <3.0.0'
-  flutter: '>=2.8.0'
+  sdk: '>=3.3.4 <4.0.0'
```
Comment (hoc081098) [inline example/pubspec.yaml:7]:
```suggestion
  sdk: '>=3.3.1 <4.0.0'
```

File: pubspec.yaml [hunk 3]
```diff
@@ -14,28 +14,28 @@ topics:
   - flutter-google-places
 
 environment:
-  sdk: '>=3.0.0 <4.0.0'
-  flutter: '>=3.10.0'
+  sdk: '>=3.3.4 <4.0.0'
```
Comment (hoc081098) [inline pubspec.yaml:17]:
```suggestion
  sdk: '>=3.3.1 <4.0.0'
```

**Pull request merged**